appcues/src/main/java/com/appcues/trait/appcues/TooltipTrait.kt

```diff
@@ -113,7 +115,7 @@ internal class TooltipTrait(
         val containerDimens = remember { mutableStateOf<TooltipContainerDimens?>(null) }
         // this is like containerDimens, but removing the size of the pointer from the width or height,
         // depending on the pointer position - used in layout calculation of desired pointer position
-        val contentDimens = remember { mutableStateOf<TooltipContainerDimens?>(null) }
+        val contentDimens = remember(targetRectInfo) { mutableStateOf<TooltipContainerDimens?>(null) }
```

Comment:
this update to the remember allows the next pointer position check to run only after the new content size is found for the new step